dataverse-webapp/src/main/webapp/metadata-importers-fragment.xhtml

```diff
@@ -85,7 +85,8 @@
                                         <c:choose>
                                             <c:when test="#{item.type eq Type.INPUT}">
                                                 <div class="col-sm-9">
-                                                    <p:inputText id="_#{item.viewId}" value="#{item.value}"/>
+                                                    <p:inputText id="_#{item.viewId}" value="#{item.value}"
+                                                        onkeypress="if (event.keyCode == 13) { nextStepCommand(); return false; }" />
```

Comment:
I think that we should use `<p:defaultCommand target="..." scope="..." />` for situations like this.

Reply:
It doesn't work in this case - I think the reason is that the "importMetadataDialog" is located inside "editMetadataForm" but it works outside form logic.

Reply:
Ok. Let's leave it as is